salt/journal/config/srv-journal-app-config-parameters.yml

```diff
@@ -17,3 +17,12 @@ parameters:
     crm_url: https://crm.elifesciences.org/crm/civicrm/
 
     disqus_domain: elifesciences-staging
+
+    # defaults for URL generation, should only be used in the CLI
+    {% if salt['elife.only_on_aws']() %}
+    router.request_context.host: {{ salt['elife.cfg']('project.full_hostname') }}
```

Comment:
Does this give the public-facing address? (eg `beta.elifesciences.org` rather than `prod--journal.elifesciences.org`)